content/channels/options/index.textile

```diff
@@ -0,0 +1,273 @@
+---
+title: Channel options
+meta_description: "Channel options customize the functionality of channels."
+languages:
+  - csharp
+  - flutter
+  - java
+  - javascript
+  - nodejs
+  - objc
+  - php
+  - python
+  - ruby
+  - swift
+---
+
+Channel options can be used to customize the functionality of channels. This includes enabling features such as "encryption":/channels/options/encryption and "deltas":/channels/options/deltas, or for a client to retrieve messages published prior to it attaching to a channel using "rewind":/channels/options/rewind.
```

Comment:
Something that is missing for me from this doc as a whole is WHY/WHEN you'd want to rewind, especially by like one message or 5 seconds. Putting in some recommendations might be helpful guidance.